service/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = ">=4.0.0"
+    }
+  }
+  required_version = ">= 0.14"
```

Comment:
this is a very outdated version of tf; please maintain the minimum tf version 1.0